compliance_checker/cf/cf.py

```diff
@@ -2903,10 +2903,12 @@ def check_climatological_statistics(self, ds):
         # variable, we need to make sure it has the attribute "climatology",
         # but not the attribute "bounds"
         meth_regex = "(?:{})".format("|".join(methods))
+
+        # find the variables with climatology attributes
         clim_containing_vars = ds.get_variables_by_attributes(
             climatology=lambda s: s is not None)
```

Comment:
I think the code here is doing the right thing, but the naming is confusing. This is identifying _climatological time axis_ variable(s) (see [CF 7.4](http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html#climatological-statistics), second paragraph), _not_ variables containing the actual climatological statistics (those would have this variable as a coordinate).

Reply:
This also implies that in the loop below, `clim_var` is a time _coordinate variable_, in the CF sense, i.e. it has only one dimension, with the same name as itself. So I don't understand the reference to "2d coordinate bounds" in line [2929](https://github.com/ioos/compliance-checker/blob/master/compliance_checker/cf/cf.py#L2929) below??